scripts/uberenv/spack_configs/linux-ubuntu18.04-broadwell/packages.yaml

```diff
@@ -2,7 +2,7 @@ packages:
   all:
     compiler: [gcc, clang]
     providers:
-      mpi: [openmpi]
+      mpi: [mpich]
```

Comment:
I was helping @samuelpmish about this file today.  A following PR is probably needed to see if you guys can come up with a common solution.  I have ideas.

Reply:
What was the issue?

Reply:
System installed packages being in different places, architectures being slightly different, versions not being updated in various ways..